scripts/test.sh

```diff
@@ -53,6 +53,10 @@ start_server() {
 }
 
 stop_server() {
+  if [ "$(uname)" = 'Linux' ]; then
```

Comment:
you're starting the server on a stop_server() function. Also.. this should probably just stop the docker container running the server.